.gitignore

```diff
@@ -1,3 +1,6 @@
+# Locally built binaries
```

Comment:
i don't think we can realistically keep this in sync. i think it's preferable to set the output location for the build to something like /tmp so it doesn't create the binary in the repo at all.

Reply:
I think the only thing we currently build is this binary, so it's fine for now? But yeah I agree that it's not great to pollute the local dir and we should eventually clean that up.

Reply:
i'm not crazy about adding application-specific names to the gitignore. what would the process be for changing the write location for the binary?

Reply:
I tweaked the build libraries to use a `./bin/` dir instead.